src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -92,6 +92,10 @@ export default class Result extends React.Component {
     );
   }
 
+  renderNote() {
+    return <Note>{t('Displaying up to 10 results')}</Note>;
```

Comment:
Where does 10 come from? Should it be configurable? At the very least shouldn't it be a constant that's used when querying?

Reply:
it's hardcoded for now (we decided we want it to always be top 10), but I agree it should be defined in one place